tripleblindmarket/private-kit
# Connect intersection logic with Exposure History screen

#### Description:
This PR introduces logic that populates the Exposur History calendar on intersection check completion.


#### How to test:
- Select an item from the list of available HAs
- wait for the intersection logic to finish
- after some time, if you have been exposed, the data should be visible in the exposure history screen
